crypt_test.go

```diff
@@ -19,7 +19,7 @@ import (
 func TestEncrypt(t *testing.T) {
 	f, err := OpenFile(filepath.Join("test", "encryptSHA1.xlsx"), Options{Password: "password"})
 	assert.NoError(t, err)
-	assert.EqualError(t, f.SaveAs(filepath.Join("test", "BadEncrypt.xlsx"), Options{Password: "password"}), "not support encryption currently")
+	assert.EqualError(t, f.SaveAs(filepath.Join("test", "TestBadEncrypt.xlsx"), Options{Password: "password"}), "not support encryption currently")
```

Comment:
I suggest to keep the spreadsheet name of them. In fact, I have a program offline that will batch verify the generated file with `Test` prefix can be opened correctly by Office applications :)